ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -148,6 +176,47 @@ class MySqlNamedBlobDb implements NamedBlobDb {
                 dataSourceFactory.getDataSource(dbEndpoint),
                 localDatacenter.equals(dbEndpoint.getDatacenter()) ? config.localPoolSize : config.remotePoolSize)));
     this.remoteDatacenters = MySqlUtils.getRemoteDcFromDbInfo(config.dbInfo, localDatacenter);
+    this.metricsRecoder = new MySqlNamedBlobDb.Metrics(metricRegistry);
+  }
+
+  private static class Metrics {
+    public final Counter namedDataNotFoundGetCount;
+    public final Counter namedDataErrorGetCount;
+    public final Counter namedDataInconsistentGetCount;
+
+    public final Counter namedDataInconsistentListCount;
+
+    public final Counter namedDataNotFoundDeleteCount;
+    public final Counter namedDataErrorDeleteCount;
+    public final Counter namedDataInconsistentDeleteCount;
+
+    public final Counter namedDataErrorPutCount;
+
+    /**
+     * Constructor to create the Metrics.
+     * @param metricRegistry The {@link MetricRegistry}.
+     */
+    public Metrics(MetricRegistry metricRegistry) {
```

Comment:
now we are on metrics here, can we add latency metrics for each sql methods?

Reply:
Sure, added. can you take another look?